### ISSUE TITLE
Branding Auth Pages

### ISSUE THREAD

**Issue by ruiengana** [JanssenProject/jans#374]
When applying branding there are internal servlets to calculate location uri of custom logo and favicon. Are these servlets applicable for Jans Auth Server? How can we configure custom logos in Jans Auth Server?

**Comment by Milton-Ch** [comment]
Hey guys, digging more on this issue, looks like we have to consider these points:
1. Name of the file should be placed inside `jansOrganization` table in DB, over there you will find two fields `jansLogoPath` and `jansFaviconPath`, please use full path.
2.  It's irrelevant whether logo or favicon are inside `/opt/gluu/jetty/jans-auth/custom/static/{logo|favicon}/` or not, the only thing is that service should have access to read that content, by preference I'd suggest to use that folder anyway because it's referenced in docs.

Please modify single entry in `jansOrganization` and try again, LMK how it goes, thanks.

**Comment by HemantKMehta** [comment]
Yes, I was also testing it with these entries for paths for logos and favicon and for me it was giving problem as there was not entry for doc_id=_ in jans.jansOrganization table. With an entry in jansOrganization for doc_id=_ and the paths of logo and favicon in respective fields it was working for me. 
We should have some way to configure these values instead of modifying them in database. Adding JSON configuration properties will be good.

**Comment by Milton-Ch** [comment]
This configuration is placed in oxTrust AFAIK, perhaps we should also enable it via jans-config-api (not sure if it is already).
https://gluu.org/docs/gluu-server/4.2/admin-guide/oxtrust-ui/#organization-settings

**Comment by HemantKMehta** [comment]
I guess at the moment it is not there in [jans-config-api](https://gluu.org/swagger-ui/?url=https://raw.githubusercontent.com/JanssenProject/jans-config-api/master/docs/jans-config-api-swagger.yaml#/Database%20-%20Sql%20configuration/get-config-database-sql).

**Comment by Milton-Ch** [comment]
Ticket created to handle it properly via `jans-config-api`. Let's wait for client confirmation in order to see whether we should fix/enhance something here. Thanks.

**Comment by pujavs** [comment]
@Milton-Ch, please chk issue [655](https://github.com/JanssenProject/jans/issues/655) and confirm if any more changes required

**Comment by pujavs** [comment]
Assigning to @Milton-Ch to take it further

**Comment by Milton-Ch** [comment]
I replied in issue https://github.com/JanssenProject/jans/issues/655 however in case that's configurable, I suppose we can close this ticket, because initial requirement was to make this configurable.
